docs/streams/stream-tasks/view-streams.md

```diff
@@ -20,7 +20,118 @@ View existing streams.
 
 1. To see more details and stream statistics, click a stream name.
 
-When you view a stream in the console, you will not see a backlog of messages. This screen does not persist, which means that when it is open, you will see new messages as they are sent. If you close the screen, any messages displayed will not be visible the next time that you open it.
+</TabItem>
+<TabItem value="py" label="Python SDK">
+
+View a list of existing streams.
+
+```python
+from c8 import C8Client
+
+""" For Python SDK, you can omit https:// part of the URL """
+BASE_URL = "play.paas.macrometa.io/"
+
+""" Connect to GDN """
+client = C8Client(
+    protocol='https',
+    host=BASE_URL,
+    port=443,
+    apikey='<your API key>', # Enter your API key
+    geofabric="<your-fabric>" # Enter your fabric name
+)
+
+streams = client.get_streams()
```

Comment:
@dlozina also somehow, I am not getting all streams, looks like only global ones are coming back.
When I use JS I get local, global and collections streams

Reply:
@Luba89 I will check interesting, thanks

Reply:
Any progress on this?

Reply:
@Luba89 it will give you all the streams.. I think you just need to update your pyC8 that's it, you seem to have an older version of pyC8 which had that issue earlier.. We had fixed that way back just updating should work for you

Reply:
It works fine for me with newer pyC8 versions